### DISCUSSION TITLE
How to remap script for specific package? dev -> start

### DISCUSSION THREAD

**Discussion by iamyoki** [vercel/turborepo#1427] [Help] [answered]
## Structure
```
root/
├── cra-app
└── packages/
    └── components
```
## What I want is
When  `turbo run dev`, remap `dev` command as `start` only in **cra-app**.

**Comment by tknickman** [comment, marked as answer]
This isn't supported at the moment. 

I would suggest either renaming `start` to `dev` in your `root/cra-app/package.json`:

```diff
  "scripts": {
-   "start": "react-scripts start",
+   "dev": "react-scripts start", 
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject"
  },
```

Or, if you still need a `start` script, you can add a second `dev` script that calls the same thing as `start` (or `start` directly if that makes more sense for your setup):

```diff
  "scripts": {
    "start": "react-scripts start",
+   "dev": "react-scripts start", 
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject"
  },
```
